importMariaDBdmpFile/config.ini

```diff
@@ -0,0 +1,25 @@
+[remote]
+# hostname of remote host from which to fetch the dump file
+host=somehost.ral.ac.uk
+# remote user. Note ssh access must be without password i.e. by key
+user=dbsync
+# path on the remote host from which to fetch the dump file
+path=/path/to/db/dump
+
+[local]
+# local 'working' directory to stage and/or inflate the dump file into.
+workDir=/tmp
+# Options file containing the password with the format -
+# [client]
+# password=myBadPassword
+mysqlOptions=/path/to/password/file
+# local directory into which successfully installed dumps are archived
+# a _goc5dump
+archiveDir=/root/importMariaDBdmpFile/lastImportedDmpFile
```

Comment:
Hmmm... not sure we need to / should archive the previously imported DB dumps.

If we did, it looks like the `archiveDir` would contain all successfully installed dumps? not just the last one?

```suggestion
archiveDir=/root/importMariaDBdmpFile/previouslyImportedDmpFiles
```

Reply:
I assumed it did (contain previously imported dumps) and there was a cron cleanup. The code goes to lengths to timestamp the archived files. No?

Reply:
ah yes, the oracle version does preserve the last imported dmp file 😛 . But there is no cron to clean it up, it looks like [the script does that itself](https://github.com/GOCDB/gocdb-failover-scripts/blob/master/importDBdmpFile/1_runDbUpdate.sh#L167).

In which case the MariaDB version should clean up after itself. Which would mean the directory name is good, but the comment should reflect that only one archived dump is kept.

Reply:
So it does! Not sure how I missed that 🤔